### ISSUE TITLE
 Does zero-shot work in minDALL-E?

### ISSUE THREAD

**Issue by SeungyounShin** [kakaobrain/minDALL-E#6]
Thanks for your amazing work!

I'm attempting zero-shot image-to-image translation, as described in the original paper, by inserting only half of the image. The outcomes are as follows. Will this problem be solved if I increase the size of the model?

<img width="945" alt="스크린샷 2021-12-15 오후 8 22 23" src="https://user-images.githubusercontent.com/20262536/146177809-3d5bfc1b-7c47-4d3f-b13a-a7384a39ab4f.png">

**Comment by LeeDoYup** [comment]
Hello @SeungyounShin, thanks for your test on the zero-shot image-to-image translation. 

As you mention, an autoregressive text-to-image generation model can conduct unseen tasks in the zero-shot manner, even though the training dataset does not include the exactly same types of text-image pairs ! 
However, the capability on the zero-shot learning increases when the model size & dataset set increase together.
Please note that the released minDALL-E is yet smaller scale model (1.3B params, 14M text-image pairs) than the original implementation of OpenAI (12B params, 250M text-image pairs).

The problem will be solved when a larger scale of model is trained on a larger number of training samples, and we will also release large scale of models.

**Comment by saehoonkim** [comment]
We really appreciate your interest in this minDALL-E project!

As in @LeeDoYup's comment, we think that a zero-shot image-to-image translation task is very challenging to minDALL-E, because this has been trained on only 14M image-text pairs. In addition, we also observe that minDALL-E has a limited understanding on visual relations, for instance, “on the top”, “under the table”, “on the left”, “on the right”, and so on. To accomplish a zero-shot image-to-image translation task, we believe that a full understanding of visual relationships is necessary. We expect that this feature emerges when a model of high capacity sees at least 100M pairs.

**Comment by afiaka87** [comment]
@saehoonkim @LeeDoYup This is an interesting confirmation. Thanks for this. Being resource constrained, I have witnessed the inability of many smaller DALL-E's (dalle-pytorch) to do zero-shot visual reasoning with a decent mask and prompt. I had hoped that the parameter/sample count wouldn't need to be so high for this feature to emerge; useful nonetheless!
